builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -321,7 +361,7 @@ func resourceVSphereVirtualMachineCreate(d *schema.ResourceData, meta interface{
 		}
 	}
 
-	if _, ok := d.GetOk("network_interface.0.ip_address"); !ok {
+	if _, ok := d.GetOk("network_interface.0.ipv4_address"); !ok {
```

Comment:
This shouldn't be accessing the internal storage format of `network_interface`, I'll follow up post-merge with a commit tweaking this and cc you so you can see what I mean. :+1: